src/Symfony/Component/Security/Core/Authentication/RememberMe/CacheTokenVerifier.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authentication\RememberMe;
+
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ */
+class CacheTokenVerifier implements TokenVerifierInterface
+{
+    private $cache;
+    private $outdatedTokenTtl;
+    private $cacheKeyPrefix;
+
+    /**
+     * @param int $outdatedTokenTtl How long the outdated token should still be considered valid. Defaults
+     *                              to 60, which matches how often the PersistentRememberMeHandler will at
+     *                              most refresh tokens. Increasing to more than that is not recommended,
+     *                              but you may use a lower value.
+     */
+    public function __construct(CacheItemPoolInterface $cache, int $outdatedTokenTtl = 60, string $cacheKeyPrefix = 'rememberme-')
+    {
+        $this->cache = $cache;
+        $this->outdatedTokenTtl = $outdatedTokenTtl;
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
```suggestion
     * {@inheritdoc}
```
per our coding standards